package insight_msgs;

message RobotPose
{
    required double roll = 1;
    required double pitch = 2;
    required double yaw = 3;
}